project/Dependencies.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+import sbt._
+
+object Dependencies {
+  val resolutionRepos = Seq("Akka Repository" at "http://repo.akka.io/releases/")
+
+  object V {
+    // Java
+    val gcpDataflow          = "1.9.0"
+    val gcpBigtable          = "0.9.4"
+    val hadoop               = "2.7.3"
+    val hbase                = "1.2.4"
+    val typesafe             = "1.3.1"
+    val jodaConvert          = "1.8.1"
+    val jodaTime             = "2.9.7"
+	val slf4j				 = "1.7.22"
+
+    // Scala
+    val specs2               = "3.7"
+    val guava                = "19.0"
+    val json4s               = "3.2.11"
+  }
+
+  object Libraries {
+    // Java
+    val gcpBigtableHBase      = "com.google.cloud.bigtable"  % "bigtable-hbase-dataflow"            % V.gcpBigtable
+    val hadoopCommon          = "org.apache.hadoop"          % "hadoop-common"                      % V.hadoop
```

Comment:
I need hadoop-common because in runtime on of its classes is needed (the Configuration class)